localized_fields/widgets.py

```diff
@@ -98,6 +99,16 @@ def build_widget_attrs(widget, value, attrs):
 
         return attrs
 
+    @staticmethod
+    def remove_if_needed(value):
+        """If the field LocalizedIntegerField is null in the DB then it must
+        be set to None so it can be represented"""
+        if isinstance(value, LocalizedIntegerValue):
```

Comment:
Will this work for other types of localized fields? Maybe this `remove_if_needed` function should be made more generic in this base class and each child class could override it and implement its own "cleaning" mechanism.